src/ui/components/ExplorerHome/ExplorerHome.js

```diff
@@ -354,22 +349,51 @@ export const ExplorerHome = ({
     </div>
   );
 
-  const formatInterval = (interval) =>
-    formatTimestamp(interval.startTimeMs, {
-      month: "short",
-      day: "numeric",
-      hour: undefined,
-      minute: undefined,
-      year: "numeric",
-    }) +
-    " to " +
-    formatTimestamp(interval.endTimeMs, {
-      month: "short",
-      day: "numeric",
-      hour: undefined,
-      minute: undefined,
-      year: "numeric",
-    });
+  const formatInterval = (interval) => {
+    const formatWithYear = (timestamp) =>
+      formatTimestamp(timestamp, {
+        month: "short",
+        day: "numeric",
+        hour: undefined,
+        minute: undefined,
+        year: "numeric",
+      });
+
+    const formatWithoutYear = (timestamp) =>
+      formatTimestamp(timestamp, {
+        month: "short",
+        day: "numeric",
+        hour: undefined,
+        minute: undefined,
+        year: undefined,
+      });
+
+    const isTodayOrLater = (someDate) => {
+      const today = new Date();
+      today.setHours(0);
+      today.setMinutes(0);
+      today.setSeconds(0);
+      today.setMilliseconds(0);
```

Comment:
Fun fact, these four lines can be replaced with `today.setHours(0, 0, 0, 0)`

Reply:
Oooh that is fun, thank you!